src/ios/app/AppleWatch.m

```diff
@@ -17,9 +21,21 @@ @implementation AppleWatch
 
 - (void) init:(CDVInvokedUrlCommand*)command {
     NSString *appGroup = [NSString stringWithFormat:@"group.%@", [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleIdentifier"]];
-
+    
+#if ( defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 90000 )
+    self.watchConnectivityListeningWormhole = MMWormholeSession.sharedListeningSession;
+    // Make sure we are activating the listening wormhole so that it will receive new messages from
+    // the WatchConnectivity framework.
+    [self.watchConnectivityListeningWormhole activateSessionListening];
+    // Initialize the wormhole using the WatchConnectivity framework's application context transiting type
+    self.wormhole = [[MMWormhole alloc] initWithApplicationGroupIdentifier:@"group.com.mutualmobile.wormhole"
```

Comment:
I *think* this should not be a fixed groupIdentifier but rather the value of `appGroup` a few some 10 lines back.